src/locales/default.json

```diff
@@ -221,7 +221,8 @@
     "voting": "Voting",
     "quorum": "Quorum",
     "type": "Type",
-    "anyType": "Any"
+    "anyType": "Any",
+    "hideAbstain": "Ignore abstain votes in basic voting results."
```

Comment:
```suggestion
    "hideAbstain": "Ignore abstain votes in basic voting results"
```

Reply:
I don't get it. What's wrong with the dot in the end? It's an imperative sentence. 😃

Reply:
It's the title of the field, not a sentence, so no dot